src/components/ClusterOverview/StorageOverview.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import { Dashboard, DashboardBody, DashboardHeader } from '../Dashboard';
+
+import { StorageDetailsConnected } from './Details/storageDetails'; // TODO: re-export
+import InventoryConnected from './Inventory/Inventory';
```

Comment:
not sure what data should be rendered here but please take into account that this card is being rewritten in https://github.com/kubevirt/web-ui-components/pull/273